### PR TITLE
fix: make sidebar radios keyboard accessible [ACC-332]

### PR DESCRIPTION
**Pull request by V-Gira**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/ACC-332" title="ACC-332" target="_blank">BugACC-332</a>  [Web] Can't reach 'Self-deleting messages' time option via keyboard
  </td></table>
  <br />
 

### Issue

- radios for self deleting message options are not keyboard accessible

### Solution
- use the former `PreferenceRadio` component for all radios

### Change
- since the component is used in different places, refactor `PreferenceRadio` to just `Radio`
- move css from less files to cs in js (taking inspiration from https://github.com/wireapp/wire-webapp/pull/14080)
- remove unused css from less files

### CODE REVIEW

File: src/script/components/Radio/Radio.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  return (
+    <div data-uie-name={uieName}>
+      {options.map(({value, label, detailLabel, isDisabled = false}) => {
+        const currentId = id + value;
+
+        return (
+          <div
+            key={value}
+            css={radioOptionStyles}
+            role="radiogroup"
```
Comment (przemvs) [inline src/script/components/Radio/Radio.tsx]: Hmm... This role should be on parent element, here for e.g. on line 49, also aria-labelledby :)

But for better accessibility you should use there: fieldset :)

File: src/script/components/Radio/Radio.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+const Radio = <T extends string | number>({
+  ariaLabelledBy,
+  name,
+  options,
+  onChange,
+  uieName = name,
+  selectedValue,
+}: RadioProps<T>) => {
+  const {current: id} = useRef(Math.random().toString(36).slice(2));
```
Comment (thisisamir98) [inline src/script/components/Radio/Radio.tsx]: let's use react's internal useId here.
https://reactjs.org/docs/hooks-reference.html#useid

**Review by thisisamir98 — APPROVED** [review]
well done, it would be great if you could add some tests for Radio.tsx
(good example is ButtonGroup.test.tsx)

File: src/script/components/Radio/RadioGroup.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+            <input
+              css={radioInputStyles(isDisabled)}
+              disabled={isDisabled}
+              tabIndex={TabIndex.FOCUSABLE}
+              type="radio"
+              id={currentId}
+              name={name}
+              value={value}
+              onChange={() => onChange(value)}
+              checked={selectedValue === value}
```
Comment (przemvs) [inline src/script/components/Radio/RadioGroup.tsx]: Move this selectedValue === value to variable, you have used it in 2 places :)

**Review by przemvs — APPROVED** [review]

**Review by przemvs — APPROVED** [review]

**Pull request merged**